config/sample.yaml

```diff
@@ -0,0 +1,30 @@
+main_domain: "futurex.sa"
```

Comment:
This is a public repo so we should restrain for using client specific sites :)

Reply:
Thanks for pointing this. Everything is much more generic now.